src/main/kotlin/no/nav/familie/ef/sak/vedlegg/DokumentinfoDto.kt

```diff
@@ -15,4 +15,7 @@ data class DokumentinfoDto(val dokumentinfoId: String,
                            val journalpostId: String,
                            val dato: LocalDateTime?,
                            val journalstatus: Journalstatus,
-                           val journalposttype: Journalposttype)
\ No newline at end of file
+                           val journalposttype: Journalposttype,
+                           val logiskeVedlegg: List<LogiskVedleggDto>? = null)
```

Comment:
Burde denne være not null, sånn att den settes til emptyList hvis `dokumentInfo.logiskeVedlegg` er null?

Reply:
Enig, der "vi" kan bestemme vil jeg ha `emptyList()` som defaultverdi :) 
Hersens `saf` som gir oss nullable lister, dessverre - men denne kan vi fintune :)

Reply:
Greit 👍